### ISSUE TITLE
Docs: Replace Latin terms in the dev_guide 

### ISSUE THREAD

**Issue by samccann** [ansible/ansible#78997]
### Summary

Our [style guide](https://docs.ansible.com/ansible/latest/dev_guide/style_guide/index.html#avoid-using-latin-phrases) notes that we should not use Latin terms (e.g, via, etc) in our documentation. This issue specifically asks to replace these with the noted English equivalents from that style guide table for all occurrences in the docs/docsite/rst/dev_guide directory. 

Use `grep -R -e 'etc\.' -e 'i\.e ' -e 'e\.g\. ' -e 'via ' -e 'vs\(\.\)\? ' -e versus --exclude-dir=locales`  in the docs/docsite/rst/dev_guide directory to find these.

List of all effected files are in a follow-on comment.

### Issue Type

Documentation Report

### Component Name

docs/docsite/rst/dev_guide/index.rst

### Ansible Version

```console
$ ansible --version
2.14
```

### Configuration

```console
# if using a version older than ansible-core 2.12 you should omit the '-t all'
$ ansible-config dump --only-changed -t all
none
```

### OS / Environment

none

### Additional Information

none

### Code of Conduct

- [X] I agree to follow the Ansible Code of Conduct

**Comment by samccann** [comment]
For the month of October, we will address hacktoberfest PRs on a first come-first merged basis as there is no way for us to assign issues to an individual contributor. Thanks for your interest in helping the Ansible documentation effort!

**Comment by samccann** [comment]
Note: there are more occurrences of Latin terms in the docs but we need to limit  the number of files in a PR so we are addressing these in small batches.

List of 27 files that need to change:
testing/sanity/future-import-boilerplate.rst:This uses Python 3 semantics for absolute vs relative imports, division, and print.  By doing this,
testing/sanity/no-unicode-literals.rst:  and a text string on python3) via a string literal.
developing_api.rst:.. note:: Ansible emits warnings and errors via the display object, which prints directly to stdout, stderr and the Ansible log.
developing_core.rst:Although ``ansible-core`` (the code hosted in the `ansible/ansible repository <https://github.com/ansible/ansible>`_ on GitHub) includes a few plugins that can be swapped out via playbook directives or configuration, much of the code there is not modular.  The documents here give insight into how the parts of ``ansible-core`` work together.
developing_inventory.rst:You can still use and write inventory scripts, as we ensured backwards compatibility via the :ref:`script inventory plugin <script_inventory>`
developing_inventory.rst:Now that you've enabled caching, loaded the correct plugin, and retrieved a unique cache key, you can set up the flow of data between the cache and your inventory using the ``cache`` parameter of the ``parse`` method. This value comes from the inventory manager and indicates whether the inventory is being refreshed (such as via ``--flush-cache`` or the meta task ``refresh_inventory``). Although the cache shouldn't be used to populate the inventory when being refreshed, the cache should be updated with the new inventory if the user has enabled caching. You can use ``self._cache`` like a dictionary. The following pattern allows refreshing the inventory to work in conjunction with caching.
developing_modules_checklist.rst:Please make sure your module meets these requirements before you submit your PR/proposal. If you have questions, reach out via the ``#ansible-devel`` chat channel (using Matrix at ansible.im or using IRC at `irc.libera.chat <https://libera.chat>`_) or the `Ansible development mailing list <https://groups.google.com/group/ansible-devel>`_.
developing_program_flow_modules.rst:    * Prior to Ansible 2.7, the module was executed via a second Python interpreter instead of being
developing_program_flow_modules.rst:sent over the wire must be executable by Python via stdin.
developing_program_flow_modules.rst:Passing arguments via stdin was chosen for the following reasons:
developing_program_flow_modules.rst:  On the other hand, if ``force`` (a boolean parameter) is set to ``true``, ``yes`` etc., both ``force_reason`` and ``force_code`` must be specified.
testing_integration_legacy.rst:The tests are invoked via a ``Makefile``.
debugging.rst:    This subcommand inserts the absolute path to ``debug_dir`` as the first item in ``sys.path`` and invokes the script using the arguments in the ``args`` file. You can continue to run the module like this until you understand the problem. Then you can copy the changes back into your real module file and test that the real module works via ``ansible`` or ``ansible-playbook``.
developing_collections_structure.rst:Ansible Collections are tested much like Ansible itself, by using the `ansible-test` utility which is released as part of Ansible, version 2.9.0 and newer. Because Ansible Collections are tested using the same tooling as Ansible itself, via `ansible-test`, all Ansible developer documentation for testing is applicable for authoring Collections Tests with one key concept to keep in mind.
developing_collections_structure.rst:When reading the :ref:`developing_testing` documentation, there will be content that applies to running Ansible from source code via a git clone, which is typical of an Ansible developer. However, it's not always typical for an Ansible Collection author to be running Ansible from source but instead from a stable release, and to create Collections it is not necessary to run Ansible from source. Therefore, when references of dealing with `ansible-test` binary paths, command completion, or environment variables are presented throughout the :ref:`developing_testing` documentation; keep in mind that it is not needed for Ansible Collection Testing because the act of installing the stable release of Ansible containing `ansible-test` is expected to setup those things for you.
developing_plugins.rst:Plugins that support embedded documentation (see :ref:`ansible-doc` for the list) should include well-formed doc strings. If you inherit from a plugin, you must document the options it takes, either via a documentation fragment or as a copy. See :ref:`module_documenting` for more information on correct documentation. Thorough documentation is a good idea even if you're developing a plugin for local use.
developing_plugins.rst:Import cache plugins using the cache_loader so you can use ``self.set_options()`` and ``self.get_option(<option_name>)``. If you import a cache plugin directly in the code base, you can only access options via ``ansible.constants``, and you break the cache plugin's ability to be used by an inventory plugin.
developing_plugins.rst:New in ansible-core 2.11, callback plugins are notified (via ``v2_playbook_on_task_start``) of :ref:`meta<meta_module>` tasks. By default, only explicit ``meta`` tasks that users list in their plays are sent to callbacks.
developing_plugins.rst:Lookup plugins pull in data from external data stores. Lookup plugins can be used within playbooks both for looping --- playbook language constructs like ``with_fileglob`` and ``with_items`` are implemented via lookup plugins --- and to return values into a variable or parameter.
testing_integration.rst:Enable PowerShell Remoting (run on the Windows host via Remote Desktop):
developing_modules_general_windows.rst:- Look for common functions in ``./lib/ansible/module_utils/powershell/`` and use the code there instead of duplicating work. These can be imported by adding the line ``#Requires -Module *`` where * is the filename to import, and will be automatically included with the module code sent to the Windows target when run via Ansible
